mllib/src/test/java/org/apache/spark/ml/feature/JavaPCASuite.java

```diff
@@ -108,7 +108,11 @@ public VectorPair call(Tuple2<Vector, Vector> pair) {
       .fit(df);
     List<Row> result = pca.transform(df).select("pca_features", "expected").toJavaRDD().collect();
     for (Row r : result) {
-      Assert.assertEquals(r.get(1), r.get(0));
```

Comment:
Probably that the previous version does not take numerical / distribution noie into account ?